python/tvm/relax/op/manipulate.py

```diff
@@ -220,3 +223,19 @@ def squeeze(x: Expr, axis: Optional[Union[int, List[int]]] = None) -> Expr:
     if isinstance(axis, int):
         axis = [axis]
     return _ffi_api.squeeze(x, axis)  # type: ignore
+
+
+@tvm.register_func("relax.run.broadcast_to")
+def torch_broadcast_to(data: tvm.nd.array, shape: tvm.nd.array) -> tvm.nd.array:
```

Comment:
likely this is no longer needed if you can lower bcast to via emit_te

Reply:
True, i dont think that lowering exists yet but as soon as it does we'll remove this function.